.github/workflows/build.yml

```diff
@@ -23,22 +23,6 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: check
-  features:
```

Comment:
Whats wrong with them? Why are you keep removing them? :)

Reply:
this was the unrebased commit from https://github.com/AluVM/aluasm/pull/5. now it is gone